student.rb

```diff
@@ -7,6 +7,6 @@ def initialize(_classroom, age, name = 'Unknown', parent_permission: true)
   end
 
```

Comment:
- [ ] In this student class, to avoid errors from the `initialize` method you need to use the correct notation for assigning `parent_permission`. Kindly change the **#super** method from this:

  ```
  super(age, name, parent_permission)
  ```
  to this:
  ```
  super(age, name, parent_permission: parent_permission)
  ```